### ISSUE TITLE
check out the results

### ISSUE THREAD

**Issue by meireles** [jesusNPL/FurDiversification#1]
@jesusNPL Can you check if the results from running the Morlon models make sense? They're all inside the DiversificationRQ_QJ/trees directory.

**Comment by jesusNPL** [comment]
Yeah, the results are correct as each scripts return a table with the mean parameter values based on the n trees from posterior distribution... By the way, the results looks awesome!

**Comment by meireles** [comment]
@jesusNPL That's great. The paleoenvironment models also finished. We'll just have to wait for Etienne's and Tanja's models now.

**Comment by jesusNPL** [comment]
I just realize that Etienne's model need to change the value for the argument "total_richness", for the complete phylogeny that value is ok, but for the nested trees need to change the value for Ntips + 10. For example for the nested tree "ftree632.nex" the arguments into the function need to be: run_DDD("ftree632.nex", total_richness = (Ntips(ftree632) + 10), number_of_trees = 1) 

Sorry about that!

**Comment by meireles** [comment]
I see @jesusNPL. Don't we need to set `total_richness` to waterver tree size we're dealing 
with plus, for example, ~ 10%?

    library("ape")
    x                 = rtree(100)  ## any given tree
    sampling_fraction = 0.9
    total_richness    = Ntip(x) + ceiling(Ntip(x) * (1 - sampling_fraction) )

**Comment by meireles** [comment]
@jesusNPL Check out the DDD test results in the debug branch 2f56aa4a9979788ae4c4988f6b6f5bb8814c8298. The problem does seem to be with the table summary function (or with DDD returning something funny). If you think those make sense I can run the code for all 100 trees.

**Comment by meireles** [comment]
@jesusNPL Tree 912 worked again but tree 768 failed with the same error: tol 1e-7 too small. (ff2b0c7329d245265fe9b4d3851c277e7e6fea3e)
This is funny because I see that new tolerances start at 1e-3 and go to 1e-6. Either that param is not being passed on or a different internal call is failing. 
I suspect that the other trees (except for the broadest one) will also fail.
Anything else to try?

**Comment by jesusNPL** [comment]
I will change the subplex optimization for the simplex one and see what happen...

**Comment by jesusNPL** [comment]
I tried to run the DDD using the subplex optimization algorithm and for the tree "ftree641" on the local desktop and after 15 minutes the computer was trying to found the convergence, but without success, so, I stop that process and changed the optimization algorithm to "simplex" and the after ~30 seconds the analyses started without any problem. Anyway I am running the analyses using only two trees and I think tomorrow we will have some results.

**Comment by meireles** [comment]
@jesusNPL "simplex" is running without any issue on the cluster. I ran 23 random samples (n cores -1) of trees 641, 768 and 912. I will run 627, 632 and 635 now, but I think this should be done by Monday!
